development/osaka/eip-7805.md

```diff
@@ -0,0 +1,9 @@
+# EIP-7805 Metrics
+
+| Name | Metric type | Usage | Sample collection event | Labels | Buckets |
+|------|-------------|-------|-------------------------|--------|---------|
+|`execution_inclusion_list_transactions_received_in_payload_total`| Counter | Total number of inclusion list transactions received in payload | On receiving payload | | |
+|`execution_inclusion_list_transactions_valid_total`| Counter | Total number of valid inclusion list transactions | On inclusion list transactions validation | | |
+|`execution_inclusion_list_transactions_invalid_total`| Counter | Total number of invalid inclusion list transactions | On inclusion list transactions validation | | |
```

Comment:
What do you think about adding metrics for gas and size? For example,

`execution_inclusion_list_transactions_valid_size_bytes_total`
`execution_inclusion_list_transactions_invalid_size_bytes_total`

`execution_inclusion_list_transactions_valid_gas_usage_total`
`execution_inclusion_list_transactions_valid_gas_limit_total`

`execution_inclusion_list_transactions_invalid_gas_usage_total`
`execution_inclusion_list_transactions_invalid_gas_limit_total`

Reply:
Also, what do you think about metrics for engine APIs such as:

`execution_engine_getInclusionListV1_requests_total`
`execution_engine_getInclusionListV1_returned_inclusion_list_transactions` (histogram)